.rubocop.yml

```diff
@@ -0,0 +1,30 @@
+AllCops:
+  NewCops: enable
+  TargetRubyVersion: 2.5.1
+
+Style/ClassAndModuleChildren:
+  EnforcedStyle: compact
+
+Style/FrozenStringLiteralComment:
+  Enabled: false
+
+Layout/LineLength:
+  Max: 180
+
+Metrics/MethodLength:
+  Max: 30
+
+Style/Documentation:
+  Enabled: false
+
+Metrics/AbcSize:
+  Enabled: false
+
+Metrics/BlockLength:
+  Enabled: false
+
+Layout/EndOfLine:
+  Enabled: false
+
+Lint/MixedRegexpCaptureTypes:
+  Enabled: false
```

Comment:
hey @cfcortes could you confirm that these are the configured `Cops` for rubocop in this repo, please

Reply:
@JuanequeX what do you think about adding this Badge to your README file?
https://github.com/rubocop/rubocop#readme-badge

And yeah, I also have the same question as Daniel, where did you get your `rubocop.yml` file, I think you can find the default file [here](https://github.com/rubocop/rubocop/blob/master/config/default.yml) and then start configuring it as you wish.

Please tell us more about your config file

Reply:
@jclopezdev  I honestly used this configuration because in the last katas one of my co-workers shared it with me, are the correct ones to use the configurations that you share with me in the link?

obviously we needed to customize it to have better results when executing rubocop

Reply:
But, in someones docs in the web are not recommend to use a specific configuration, they recommend remove it an use only the default configurations by the gem?

Reply:
> hey @cfcortes could you confirm that these are the configured `Cops` for rubocop in this repo, please

@dannharo we do not have something pre-defined here. The only requirement is to use Rubocop, what would be your advise on this?

Reply:
I just wanted to know that

Reply:
so we approve this PR?